portfolio/src/main/webapp/script.js

```diff
@@ -37,11 +37,29 @@ async function showServerTime() {
   dateContainer.innerText = textFromResponse;
 }
 
-async function addRandomString() {
+async function getRandomString() {
+    
   const responseFromServer = await fetch('/string');
+  console.log("Fetch funcionó correctamente");
   const textFromResponse = await responseFromServer.text();
 
-  const dateContainer = document.getElementById('string-container');
-  dateContainer.innerText = textFromResponse;
+  const strings = document.getElementById('string');
+  strings.innerText = textFromResponse;
+
+  // The json() function returns an object that contains fields that we can
+  // reference to create HTML.
+  const str = await responseFromServer.json();
+
+  const statsListElement = document.getElementById('string');
+  statsListElement.innerHTML = '';
+
+  statsListElement.appendChild(str.json1);
```

Comment:
This doesn't look as I would expect it to and I'm having trouble understanding how this works. Is it working?